modin/pandas/test/dataframe/test_default.py

```diff
@@ -472,11 +484,21 @@ def test_mad_level(level):
     "value_vars", [lambda df: df.columns[-1], lambda df: df.columns[-4:], None]
 )
 def test_melt(data, id_vars, value_vars):
+    if StorageFormat.get() == "Hdk":
+        # Drop NA and sort by all columns to make sure the order
+        # is identical to Pandas.
```

Comment:
Should we have an issue for this to fix or to put some notes in docs? Can be done in a separate PR.

Reply:
It's mentioned in the documentation, that sort_values() does not guarantee the index order.